services/service-core/src/auth/keycloak.service.ts

```diff
@@ -43,4 +78,32 @@ export class KeycloakService {
       username: decodedToken['preferred_username']
     };
   }
+
+  async editUser(id: string, userInput: UpdateUser) {
+    const accessToken = await this.getAdminToken();
+    if (
+      userInput.firstName?.trim() === '' ||
```

Comment:
What if tomorrow we add two more params? Will we keep growing this statement? There could be many ways how to approach this but the best one is to start using `joi`. It's too big for the scope if this task but I will add it as one more story under tech debt